amr-wind/equation_systems/icns/source_terms/CoriolisForcing.cpp

```diff
@@ -80,7 +80,14 @@ void CoriolisForcing::operator()(
     const auto sinphi = m_sinphi;
     const auto cosphi = m_cosphi;
     const auto corfac = m_coriolis_factor;
-    const auto S = m_S;
+
+    amrex::Real S;
+    if (m_S == true) {
+        S = 1.0
+    } else {
+        S = 0.0
+    }
+
```

Comment:
Thanks, just a small suggestion, this can be an one-liner like this
`amrex::Real S = (m_three_dimensional_forcing==true) ? 1.0 : 0.0;`. I would also recommend changing m_S to something that makes more sense when you read the code for the first time.

Reply:
I have made the changes thanks for the suggestion.